otherlibs/systhreads/st_stubs.c

```diff
@@ -59,13 +59,53 @@
 /* Max computation time before rescheduling, in milliseconds */
 #define Thread_timeout 50
 
+typedef int st_retcode;
+
+/* OS-specific initialization */
+static int st_initialize(void)
+{
+  return 0;
+}
```

Comment:
I think this could probably have been removed in 5.0 itself - if it's not needed as part of the subsequent removal, I think it can just go (in 4.14 it had a purpose)